sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/SendSmsToGroup.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.sms.samples.quickstart;
+
+import com.azure.communication.sms.SmsClient;
+import com.azure.communication.sms.SmsClientBuilder;
+import com.azure.communication.sms.models.SmsSendOptions;
+import com.azure.communication.sms.models.SmsSendResult;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.Context;
+
+import java.util.Arrays;
+
+public class SendSmsToGroup {
+    public static void main(String[] args) {
+        String endpoint = System.getenv("AZURE_COMMUNICATION_ENDPOINT");
+        String accessKey = System.getenv("AZURE_COMMUNICATION_KEY");
+        String phone_number = System.getenv("COMMUNICATION_PHONE_NUMBER");
+        AzureKeyCredential keyCredential = new AzureKeyCredential(accessKey);
+        SmsClient smsClient = new SmsClientBuilder()
+            .endpoint(endpoint)
+            .credential(keyCredential)
+            .buildClient();
+
+        SmsSendOptions options = new SmsSendOptions();
+        options.setDeliveryReportEnabled(true);
+        options.setTag("Marketing");
+
```

Comment:
nit: extra line

Reply:
Unresolved this convo as I still see extra lines (28 and 29)

Reply:
please check after the last commit